tests/functional/adapter/iceberg/test_iceberg_support.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from tests.functional.adapter.iceberg import fixtures
+from dbt.tests import util
+
+
+@pytest.mark.skip_profile("databricks_cluster")
+class TestIcebergTables:
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "first_table.sql": fixtures.basic_table,
+            "iceberg_table.sql": fixtures.basic_iceberg,
+            "table_built_on_iceberg_table.sql": fixtures.ref_iceberg,
+        }
+
+    def test_iceberg_refs(self, project):
+        run_results = util.run_dbt()
+        assert len(run_results) == 3
+
+
+@pytest.mark.skip_profile("databricks_cluster")
+class TestIcebergSwap:
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {"first_model.sql": fixtures.basic_view}
+
+    def test_iceberg_swaps(self, project):
```

Comment:
Test that we can replace a view with iceberg table, and swap to incrementing that table with incremental.  If tblproperties weren't so hard to manage, we would also remove iceberg when the user removes the table_format config; however, here we do not.  Will need to address when I write the docs.